src/pages/academy/index.js

```diff
@@ -381,19 +378,15 @@ const DerivBlog = ({ data }) => {
                 <Carousel has_autoplay autoplay_interval={6000} {...settings}>
                     {homepage_banner_data.map((page_data) => {
                         return (
-                            <StyledLink
-                                key={page_data.id}
-                                href={page_data.link}
-                                rel="noopener noreferrer"
-                                target="_blank"
-                            >
+                            <Flex key={page_data.id}>
                                 <Hero
                                     imageData={page_data.desktop_hero.imageFile}
                                     imageAlt={page_data?.desktop_hero?.description}
                                     title={page_data.heading}
                                     description={page_data.sub_heading}
+                                    href={page_data.link}
```

Comment:
We also need to pass the CTA button text coming from the API. We'll need to create the field for the CMS.